pkg/cluster-runtime/installer.go

```diff
@@ -196,11 +196,11 @@ func (i *Installer) Install() error {
 
 	appInstaller := NewAppInstaller(i.infraDriver, i.Distributor, extension)
 
-	if err = appInstaller.LaunchClusterImage(master0, cmds); err != nil {
+	if err = appInstaller.Launch(master0, cmds); err != nil {
 		return err
 	}
 
-	return nil
+	return appInstaller.save(common.GetDefaultApplicationFile())
```

Comment:
I'm more confused here。the logic that saving `DefaultApplicationFile` to cluster should belongs to appinstaller `s Launch, why we do it splitly  ?